src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/allcom_db
 spring.datasource.username=root
 server.port=8080
-spring.liquibase.change-log=classpath:/db/changelog/db.changelog-master.xml
\ No newline at end of file
+spring.liquibase.change-log=classpath:/db/changelog/db.changelog-master.xml
+photo.upload.path=/Users/dariyal/ProjectAIT/allcom-backend/images
```

Comment:
We don’t need this path in the working environment settings; for development, we can use the project configuration settings and fill in the environment variables there